crates/storage/db/src/implementation/mdbx/cursor.rs

```diff
@@ -158,11 +158,25 @@ impl<K: TransactionKind, T: Table> DbCursorRO<T> for Cursor<K, T> {
 }
 
 impl<K: TransactionKind, T: DupSort> DbDupCursorRO<T> for Cursor<K, T> {
+    /// Returns the previous `(key, value)` pair of a DUPSORT table.
```

Comment:
```suggestion
    /// Returns the previous kv-pair of a DUPSORT table.
```
just fyi, also usually written like this for simplicity

Reply:
Yep - nicer. Just followed existing comment style.